services/venice-router/src/main/java/com/linkedin/venice/router/throttle/ReadRequestThrottler.java

```diff
@@ -89,6 +89,7 @@ public ReadRequestThrottler(
     this.storeQuotaCheckTimeWindow = storeQuotaCheckTimeWindow;
     this.stats = stats;
     this.maxRouterReadCapacity = maxRouterReadCapacity;
+    this.zkRoutersManager.subscribeRouterCountChangedEvent(this);
```

Comment:
Can we get rid of `RoutersClusterConfig` and the related code as well?
This config and related listeners are not being used in prod.

Reply:
Remove the reference to `RoutersClusterConfig`